clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -336,7 +336,7 @@ class DockerRegistryClient {
           registryService.getCatalog(paginateSize, token, userAgent)
       }, "_catalog")
     } catch (Exception e) {
-      log.warn("Error encountered during catalog of $path" + e.getMessage())
+      log.warn("Error encountered during catalog of $path: " + e.getMessage())
```

Comment:
```suggestion
      log.warn("Error encountered during catalog of $path", e)
```

that way you get the whole stack trace, etc

Reply:
Fixed in a2a414c as well.